cdk/.eslintrc.js

```diff
@@ -0,0 +1,20 @@
+module.exports = {
+  root: true,
+  env: {
+    node: true,
+    jest: true,
+  },
+  extends: ["@guardian/eslint-config-typescript"],
+  parserOptions: {
+    ecmaVersion: 2020,
+    tsconfigRootDir: __dirname,
+    sourceType: "module",
+    project: ["./tsconfig.eslint.json"],
+  },
+  plugins: ["@typescript-eslint"],
+  rules: {
+    "@typescript-eslint/no-inferrable-types": 0,
+    "import/no-namespace": 2,
+  },
+  ignorePatterns: ["**/*.js", "node_modules"],
+};
```

Comment:
@jacobwinch @akash1810 

As mentioned in [support frontend PR](https://github.com/guardian/support-frontend/pull/3765) having every project managed the cdk build toolchain on all associated files (`cdk/.eslintrc.js`, `cdk/.editorconfig`, `cdk/.gitignore`, `cdk/jest.config.js`, `cdk/package.json`, script files, etc...) is not desirable.

Could you look into this before we expand and grow our GuCDK usage?

Reply:
Just for completeness, here is the card: https://trello.com/c/19LxrUeL/1291-gucdk-zero-config.

Reply:
Agree this is something we should prioritise soon.

Reply:
Thanks!